blank_project/src/_wallet

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+</head>
+<body style="background: #fff; margin-top: 3em">
+  <div>Please run the following command in shell, then enter account id here.
+  </div>
+  <div>
+      <code id="shell-command"></code>
+  </div>
+  <input type="text" id="accountId" name="accountId" placeholder="Account id"></input>
+  <button type="button" onClick="done()">done</button>
+  <script src="https://cdn.jsdelivr.net/gh/nearprotocol/nearlib@82eed3127d56984e850cf1581344bc098b9f83ab/dist/nearlib.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/js-cookie@2/src/js.cookie.min.js"></script>
+  <script src="./config.js"></script>
+  <script>
+    const currentUrl = new URL(window.location.href);
+    const message = `near create_account {newAccountId} --masterAccount {masterAccountId} --publicKey ${currentUrl.searchParams.get('public_key')} --initialAmount 1000`;
```

Comment:
`--masterAccount {masterAccountId}` – my understanding this shouldn't normally be needed cause obtained from nearcore?

Reply:
It's a good option to spell out in case they want to use a different account to transfer funds from.

Reply:
then it makes sense to have 2 different suggestions – one with all possible params, and one minimal one which would easy to copy-paste and run (editing command line in multiple places ins't fun).

my understanding is that this should be enough for most cases:
```
`near create_account --publicKey ${currentUrl.searchParams.get('public_key')} {newAccountId}`
```

Reply:
@potatodepaulo  dev UX question: 2 commands to choose from with an easy copy-paste option vs 1 command that needs to be tweaked -- what do you think is better?

Reply:
For context, it's a diff between these 2 (roughly)
<img width="1352" alt="Screen Shot 2019-07-30 at 4 16 06 PM" src="https://user-images.githubusercontent.com/1320689/62172100-05bceb00-b2e6-11e9-8eba-6e8bd599946f.png">
<img width="1342" alt="Screen Shot 2019-07-30 at 4 19 13 PM" src="https://user-images.githubusercontent.com/1320689/62172101-05bceb00-b2e6-11e9-8b69-7e491cff3d7d.png">

Reply:
Adding two cents here: 
The one line is much less confusing to an outsider coming in. We should always reduce the number of choices a user must make. I think the long term solution is that the command is fully generated, but this is a step in that direction.

Reply:
@vgrichina in this specific case, I think here is the solution: 
We show the long command so that the user doesn't need to think about what they're copying and pasting. Even in the cases this is more than needed, that use case is still not going to break.
To address your point we should add a link here on this page that says "What am I seeing?" that takes them into the shell documentation (Or the new accounts documentation we're working on). This is a good compromise, since savvy users will be able to figure out their own commands.